docs/references/protocol/ledger-data/ledger-entry-types/permissioneddomain.md

```diff
@@ -0,0 +1,103 @@
+---
+seo:
+    description: A PermissionedDomain ledger entry represents a Permissioned Domain, which is used to limit access to other features.
+labels:
+  - Compliance
+  - Permissioned Domains
+---
+# PermissionedDomain
+[[Source]](https://github.com/XRPLF/rippled/blob/master/include/xrpl/protocol/detail/ledger_entries.macro#L451-L461 "Source")
+
+A `PermissionedDomain` ledger entry describes a single [permissioned domain][] instance. You can create a permissioned domain by sending a [PermissionedDomainSet transaction][].
+
+_(Requires the [PermissionedDomains amendment][] {% not-enabled /%})_
+
+
+## Example {% $frontmatter.seo.title %} JSON
+
+```json
+{
+  "LedgerEntryType": "PermissionedDomain",
+  "Fee": "10",
+  "Flags": 0,
+  "Owner": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+  "OwnerNode": "0000000000000000",
+  "Sequence": 390,
+  "AcceptedCredentials": [
+    {
+        "Credential": {
+            "Issuer": "ra5nK24KXen9AHvsdFTKHSANinZseWnPcX",
+            "CredentialType": "6D795F63726564656E7469616C"
+        }
+    }
+  ],
+  "PreviousTxnID": "E7E3F2BBAAF48CF893896E48DC4A02BDA0C747B198D5AE18BC3D7567EE64B904",
+  "PreviousTxnLgrSeq": 8734523,
+  "index": "3DFA1DDEA27AF7E466DE395CCB16158E07ECA6BC4EB5580F75EBD39DE833645F"
+}
+```
+
+<!-- TODO: use a real example above -->
+
+## {% $frontmatter.seo.title %} Fields
+
+In addition to the [common fields](../common-fields.md), {% code-page-name /%} entries have the following fields:
+
+| Field                 | JSON Type            | [Internal Type][] | Required? | Description  |
+|:----------------------|:----------|:------------------|:----------|--------------|
+| `AcceptedCredentials` | Array                | Array             | Yes       | A list of 1 to 10 [**Accepted Credentials objects**](#acceptedcredentials-objects) that grant access to this domain. The array is stored sorted by issuer. |
```

Comment:
I'd put the word _objects_ outside the link, as in
\[Accepted Credentials\](#acceptedcredentials-objects) objects....

Also, I can't find another example where we put link text in bold.

(I get that you're using the name of the heading you're linking to, but the context is the objects, not the heading.)